.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 /dist/
 /cli/embedded-app/dist/**/*
 doc/
+.vscode
```

Comment:
Add a new line at the end, just to make the git happier on processing of CR and CRLF.